content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -89,49 +96,82 @@ For the Quickstart, create a testing [COSE Key](https://cose-wg.github.io/cose-s
   openssl ecparam -name prime256v1 -genkey -out $SIGNING_KEY
   ```
 
-## Generate a Payload
+## Generating a Statement
 
-Create any payload you wish to register on DataTrails. JSON based payloads are indexed for query capabilities.
+1. Create a simple json statement
 
-{{< note >}}
-The current version of the DataTrails SCITT API is limited to JSON payloads.
-This will be relaxed in a future release
-{{< /note >}}
+    ```bash
+    cat > $STATEMENT_FILE <<EOF
+    {
+        "build-server": "22",
+        "test-results": "passed",
+        "compliance-22": "compliant"
+    }
+    EOF
+    ```
+
+1. Optionally save the statement in the [DataTrails Events Attachments API](/developers/api-reference/events-api/#adding-attachments)
```

Comment:
I thought we were going to punt the attachments api?

I think the minimum the sample script does is take a file, hash it and add that to the payload, while any file metadata is stored in the protected header?

Reply:
I think this is  ok though ? We are just saying, "we have an attachments api, and you can use it with scitt like this if you want" ?

Reply:
I think this is probably going to be the first touch experience for alot of developers that want to try/use scitt with datatrails.

I think it maybe better to have a separate how-to for detached statements and how to deal with them, using the python sample scripts (that will be written). Then that how-to can go into more detail about storage (if we want) but talking about what goes into the payload and what goes in the protected headers...etc

I think its very good having a statement saying something along the lines of `if your statement is more complex than a small amount of json, see this how-to ... (insert link)`

Reply:
latest change reflects us keeping the attached and detached scenarios. I've kept as one doc for now to ease our viewing. 
I can split into separate docs if needed as it becomes functional.